misc/cross-compile/cross_compile_readme.md

```diff
@@ -0,0 +1,30 @@
+# cross-compile-arm64Mac-to-x86_64Linux.sh
+
+Modify variables in the script if needed!
+
+## What it does
+- Cross-compiles Godot Mono C# projects.
+- Runs on macOS (Apple Silicon).
+- Outputs Linux x86_64 binaries (e.g., SteamOS/ArchLinux).
+
+## Requirements
+- Docker installed (`brew install docker`).
+- Godot Mono project ready to export.
+
+## How to use
+1. Place `cross_compile.sh` in project root.
+2. Open terminal, go to project root.
+3. Run: `bash cross_compile.sh`.
+4. Find output in `exports/<YourGodotMonoAppName>_linux_x86_64`.
+5. Copy the files and folders in the export folder to your target machine e.g. Steamdeck.
+6. Please remember to `chmod +x <GodotExecutable>` for execution permissions.
```

Comment:
Now I understand. This has nothing to do with the Godot buildsystem. Unfortunately, it doesn't belong in the repo, IMHO.